livekit-api/src/signal_client/mod.rs

```diff
@@ -68,11 +68,18 @@ pub enum SignalError {
 pub struct SignalOptions {
     pub auto_subscribe: bool,
     pub adaptive_stream: bool,
+    pub sdk: String,
+    pub sdk_version: Option<String>,
```

Comment:
let's put both of them to Option by default, just for consistency
```suggestion
    pub sdk: Option<String>,
    pub sdk_version: Option<String>,
```

Reply:
hmm I don't really want to... there wasn't an existing case where the sdk wasn't set already